.github/workflows/ci.yml

```diff
@@ -100,6 +100,9 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
+      - name: Build packages
```

Comment:
Without this, the type-check fails as the `@ai-sdk/vercel` package hasn't been built. I'm not sure why this package is the first to trigger this failure. It seems logical that we would need the packages built for the type-check to succeed. Perhaps the existing packages are/were cached by some other build or action, and we've been getting lucky?

Reply:
the type check does not need pkgs to be built afaik